mongo/tests/mocked_api.py

```diff
@@ -35,9 +35,17 @@ def __init__(self, db_name, coll_name):
             with open(os.path.join(HERE, "fixtures", "system.replset"), 'r') as f:
                 content = json.load(f, object_hook=json_util.object_hook)
                 self.find_one = MagicMock(return_value=content)
-        else:
-            with open(os.path.join(HERE, "fixtures", f"indexStats-{coll_name}"), 'r') as f:
-                self.aggregate = MagicMock(return_value=json.load(f, object_hook=json_util.object_hook))
+        # else:
+        #     with open(os.path.join(HERE, "fixtures", f"indexStats-{coll_name}"), 'r') as f:
+        #         self.aggregate = MagicMock(return_value=json.load(f, object_hook=json_util.object_hook))
+
+    def aggregate(self, pipeline, session=None, **kwargs):
```

Comment:
I don't get what's going on with the aggregate method. It is defined both as a property and a function.
Otherwise lgtm! 👍

Reply:
i've removed `aggregate` from the `__init__`. It's supposed to be a mocked method of the `MockedCollection` class. Prior to this PR, the only place that uses aggregate on a collection was index stats collection with `$indexStats`. That's why it worked as a mocked property from `__init__`. In this PR, we now have 2 places $indexStats and $collStats using aggregation on collections, it's more clear to properly define `aggregate` as a mocked method.